ioc/src/test/java/tw/waterball/survive/ioc/IocContainerTest.java

```diff
@@ -0,0 +1,55 @@
+package tw.waterball.survive.ioc;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import tw.waterball.survive.ioc.exceptions.CycleDependencyException;
+import tw.waterball.survive.ioc.testclasses.acyclicclasses.A;
+import tw.waterball.survive.ioc.testclasses.cyclicclasses.F;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+public class IocContainerTest {
+
+    private static IocContainer iocContainer;
+
+    @BeforeClass
+    public static void init() {
+        iocContainer = new IocContainer();
+    }
+
+    @Test
+    public void scanAcyclicPackage() {
+        iocContainer.scan("tw.waterball.survive.ioc.testclasses.acyclicclasses");
```

Comment:
I recommend renaming the package `acyclicclasses`  to `acyclic`, this is clear enough since there is a context.